prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -416,6 +417,11 @@ def prompt_processor() -> Any:
                             LogLevel.ERROR,
                         )
                         structured_output[output[PSKeys.NAME]] = None
+                        if isinstance(
+                            llm.get_llm(adapter_instance_id=adapter_instance_id),
+                            NoOpCustomLLM,
+                        ):
+                            structured_output[output[PSKeys.NAME]] = answer
```

Comment:
@harini-venkataraman why do we add this check for `NUMBER` type alone? Please add a comment as well

Reply:
Added a dev comment for this